session.go

```diff
@@ -260,6 +262,15 @@ func WithServer(addr string) ConnectOption {
 	}
 }
 
+// WithTLSConfig allows customization of the TLS connection made from the agent
+// to the ngrok service. Customization is applied after the [WithServer] and
+// [WithCA] options are applied.
```

Comment:
We could also mention that "WithCA" (and I think "WithServer" too, right?) have no impact if this is set.

Reply:
yeah, `WithCA` and `WithServer` (and `WithRegion`) do affect what gets put in the "base" tls config!